node.js

```diff
@@ -49,7 +49,42 @@ module.exports = {
     'space-before-function-paren': off,
     'keyword-spacing': error,
     'eol-last': [ error, always ],
-    'brace-style': [ error, '1tbs', { allowSingleLine: true } ],
-    'curly': [ error, all ]
+    'brace-style': [ error, '1tbs', { allowSingleLine: false } ],
+    'curly': [ error, all ],
+    'block-spacing': [ error,  always ],
+    'indent': [ error, 2, { SwitchCase: 1 } ],
+    'no-tabs': error,
+    'spaced-comment': [ error, always ],
+    'array-callback-return': warn,
+    'eqeqeq': [ error, always, { null: 'ignore' } ],
+    'no-else-return': [ warn, { allowElseIf: false } ],
+    'yoda': error,
+    'no-throw-literal': error,
+    'no-self-compare': error,
+    'no-return-assign': error,
+    'no-loop-func': error,
+    'valid-jsdoc': warn,
+    'for-direction': error,
+    'no-buffer-constructor': error,
+    'no-new-require': error,
+    'no-path-concat': error,
+    'comma-spacing': error,
+    'comma-style': error,
+    'computed-property-spacing': [ error, never ],
+    'consistent-this': [ error, 'self' ],
```

Comment:
If you're using a `this` assignment for context, use it consistently. Also, call it `self`, which cannot be used to mean anything else.
https://eslint.org/docs/rules/consistent-this
```js
// Old
function x() {
  const this2 = this
  this.doSomething()
}
// New
function x() {
  const self = this
  self.doSomething()
}
```
*Not auto-fixable but we should be using arrow functions to avoid context problems*

Reply:
Would prefer to use something other than `self` if we could provide a meaningful name in the context.